apps/backend/.env.example

```
@@ -1 +1,3 @@
 DATABASE_URL=postgres://cvsa:password@localhost:5432/cvsa
+MEILI_MASTER_KEY=0712041211241099
+MEILI_API_URL=http://127.0.0.1:7700
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MEILI_API_URL key should go before the MEILI_MASTER_KEY key

(UnorderedKey)